src/bin/pgcopydb/cli_clone_follow.c

```diff
@@ -249,6 +249,12 @@ clone_and_follow(CopyDataSpec *copySpecs)
 		exit(EXIT_CODE_INTERNAL_ERROR);
 	}
 
+	if (!copydb_fetch_schema_and_prepare_specs(copySpecs))
+	{
+		/* errors have already been logged */
+		exit(EXIT_CODE_INTERNAL_ERROR);
+	}
+
```

Comment:
I believe sub-processes also call that same function, a comment here explaining why it makes sense to call early in the main process would make sense. I suppose it's all about preparing the caches?